package.json

```
@@ -1,18 +1,30 @@
 {
-  "name": "codemod",
+  "name": "@strapi/codemods",
   "version": "1.0.0",
   "main": "index.js",
   "license": "MIT",
+  "bin": {
+    "strapi-codemods": "bin/cli.js"
+  },
+  "files": [
+    "bin/",
+    "lib/"
+  ],
   "scripts": {
     "test": "jest"
   },
   "dependencies": {
     "axios": "^0.21.2",
+    "commander": "^8.2.0",
+    "consola": "^2.15.3",
     "execa": "^5.1.1",
     "fs-extra": "^10.0.0",
+    "inquirer": "^8.2.0",
+    "inquirer-fuzzy-path": "^2.3.0",
     "jscodeshift": "^0.11.0",
```

I think bumping this gets rid of that channel closed error I had 🤷‍♂️ 
```suggestion
    "jscodeshift": "^0.13.0",
```

---

Done in latest commit